src/throttledStore.tsx

```diff
@@ -64,11 +216,58 @@ export const createThrottledStore = (
         notifySubscribers()
     }
 
-    return _.defaults(
-        {
-            subscribe,
-            flush
+    const dispatch: Dispatch<AnyAction> = action => {
+        resetPendingNotifications()
+        const dispatchResult = store.dispatch(action)
+        return dispatchResult
+    }
+
+    const result: PrivateThrottledStore = {
+        ...store,
+        subscribe,
+        dispatch,
+        flush,
+        broadcastNotify: onBroadcastNotify,
+        observableNotify: onObservableNotify,
+        resetPendingNotifications
+    }
+
+    resetPendingNotifications()
+    return result
+}
+
+export const createObservable = <TState, TSelector>(
+    shell: Shell,
+    uniqueName: string,
+    selectorFactory: (state: TState) => TSelector
+): PrivateObservableState<TState, TSelector> => {
+    const subscribersSlotKey: SlotKey<StateObserver<TSelector>> = {
+        name: uniqueName
+    }
+    const observersSlot = shell.declareSlot(subscribersSlotKey)
+    let cachedSelector: TSelector | undefined
+
+    const getOrCreateCachedSelector = (): TSelector => {
+        if (cachedSelector) {
+            return cachedSelector
+        }
```

Comment:
We can use `_.memoize` the also returns a function than has `.clear()`

Reply:
Yeah - but since this code is trivial, it looks more readable to me when written in the primitive way.